src/components/component-playground.jsx

```diff
@@ -180,15 +179,41 @@ module.exports = React.createClass({
     </ul>;
   },
 
+  _renderPreview: function() {
+    return <div ref="previewContainer" className={this._getPreviewClasses()}>
+      {this.loadChild('preview')}
+    </div>
+  },
+
   _renderContentFrame: function() {
     return <div ref="contentFrame" className={this._getContentFrameClasses()}>
-      <div ref="previewContainer" className={this._getPreviewClasses()}>
-        {this.loadChild('preview')}
-      </div>
-      {this.props.editor ? this._renderFixtureEditor() : null}
+      {this.props.editor ? this._renderSplitPane() : this._renderPreview()}
     </div>
   },
 
+  _renderSplitPane: function() {
+    return <SplitPane
+      split={this._getOrientationDirection()}
+      ref='editorPreviewSplitPane'
+      defaultSize={localStorage.getItem('splitPos')}
```

Comment:
`defaultSize` is undefined on initial view. What if the browser doesn't have support for `localstorage`?